client/dist/index.html

```diff
@@ -5,7 +5,6 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Adventure Time</title>
     <link rel="stylesheet" href="styles.css"></link>
-    <!-- // using W3 stylesheet for progressbar -->
     <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
```

Comment:
Would be fab to get the relevant styles set up ourselves so we're not pulling in another file, but it's non-breaking

Reply:
Will do. That is on my to-do list. Alex had a similar comment earlier.